framework/include/media/MediaPlayer.h

```diff
@@ -90,7 +91,7 @@ class MediaPlayer
 	 * @return The result of the create operation
 	 * @since TizenRT v2.0
 	 */
-	player_result_t create();
+	player_result_t create(stream_info_t *stream_info = nullptr);
```

Comment:
Do not change the api, 
add the create api containing the argument ~?

Reply:
Added setStreamInfo api